src/pages/iou/IOUModal.js

```diff
@@ -138,11 +143,15 @@ class IOUModal extends Component {
     }
 
     componentDidMount() {
-        PersonalDetails.fetchLocalCurrency();
+        this.fetchData();
         IOU.setIOUSelectedCurrency(this.props.myPersonalDetails.localCurrencyCode);
     }
 
     componentDidUpdate(prevProps) {
+        if (prevProps.network.isOffline && !this.props.network.isOffline) {
+            this.fetchData();
+        }
```

Comment:
Needs to be updated.

Reply:
Is the condition wrong or you want an early `return`?
I chose not return because there were codes in the `componentDidUpdate()`

Reply:
Got it. Sorry I missed there was more in this method. Thanks, will resolve. Still need to update the test steps in this PR.